HW_2/task_4.py

```diff
@@ -0,0 +1,31 @@
+"""
+Задайте список из N элементов, заполненный целыми числами из промежутка [-N, N].
+Найдите произведение элементов на индексах, хранящихся в файле indexes.txt (в одной строке один индекс).
+Решение должно работать при любом натуральном N.
+
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+"""
+# path = 'indexes.txt'
+# data = open(path, 'r')
+# for line in data:
+#     print(line)
+# data.close()
+N = int(input('Enter N: '))
+import random
+
+rnd_list = []
+for i in range(0, N):
```

Comment:
Ноль можно не указывать первым аргументом.
```suggestion
for i in range(N):
```